pep-0720.rst

```diff
@@ -0,0 +1,939 @@
+PEP: 720
+Title: Cross-compiling Python packages
+Author: Filipe Laíns <lains@riseup.net>
+PEP-Delegate:
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 01-Jul-2023
+Python-Version: 3.12
```

Comment:
Actually, we don't need it for an informational PEP:
```suggestion
```

https://peps.python.org/pep-0001/#pep-header-preamble says:

> Standards Track PEPs will typically have a Python-Version header which indicates the version of Python that the feature will be released with. Standards Track PEPs without a Python-Version header indicate interoperability standards that will initially be supported through external libraries and tools, and then potentially supplemented by a later PEP to add support to the standard library. Informational and Process PEPs do not need a Python-Version header.

Reply:
I put 3.12 because that's the version up-to which this PEP covers. Thing are still too up in the air for 3.13, and my plan is to actually introduce some changes, which I think will likely be able to make it to 3.13. I think keeping Python version is useful to make it easier to flag that the information presented might be outdated.

What do you think? Do you think it's still better to remove the version field? If not, do you still think it should be 3.13?